pkg/security/api/validation/validation_test.go

```diff
@@ -151,15 +151,15 @@ func TestValidatePodSecurityPolicyReview(t *testing.T) {
 				},
 			},
 		},
-		`spec.serviceAccountNames[0]: Invalid value: "my bad sa": must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')`: {
+		`spec.serviceAccountNames[0]: Invalid value: "my bad sa": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')`: {
```

Comment:
You could squash all the changes fixing this test string into a single commit. At least `fix-test: pkg/image/api/validation/validation_test.go` and `fix-test: pkg/api`. I'd also squash `fix-test: pkg/deploy/api/validation/validation_test.go error output` with it as well.